pkg/odo/cli/remove/binding/binding.go

```diff
@@ -86,7 +86,7 @@ func NewCmdBinding(name, fullName string) *cobra.Command {
 		},
 	}
 	bindingCmd.Flags().String(backend.FLAG_NAME, "", "Name of the Binding to create")
-	clientset.Add(bindingCmd, clientset.BINDING, clientset.FILESYSTEM)
+	clientset.Add(bindingCmd, clientset.REMOVE_BINDING, clientset.FILESYSTEM)
```

Comment:
I'm not comfortable with this solution. If the problem is to define if KUBERNETES must be nullable or not for a specific command, we need to find a way to specify this directly

Reply:
Intuitively, I think KUBERNETES and KUBERNETES_NULLABLE should be mutually exclusive, but in cases where both are needed, we could only return an error if KUBERNETES_NULLABLE was not defined. WDYT?

Reply:
@valaparthvi Yes, that seems a very good idea to me